Vagrantfile

```diff
@@ -96,7 +105,6 @@ Vagrant.configure("2") do |config|
     v.customize ["modifyvm", :id, "--nested-hw-virt", "on"]
     # Use the recommended paravirtualization interface for windows (hyperv) - requires VirtualBox 6
     v.customize ["modifyvm", :id, "--paravirtprovider", "hyperv"]
-    override.vm.network :private_network, ip: "192.168.59.90", gateway: "192.168.56.1"
```

Comment:
We removed this line over here, because we didn't take the time to validate override-order yet, i.e. how the code would behave when it would have been left in place. Do you think our amendments re. "per-box-IP" would still take precedence?

However, it might introduce a regression because all the NON-"out-of-the-box" boxes will now be invoked without this configuration setting.

Please let us know if this detail should be addressed and verified beforehand.

Reply:
Is GH-89 possibly related?

/cc @gabyx